src/guides/v2.3/javascript-dev-guide/javascript/requirejs.md

```diff
@@ -62,6 +62,28 @@ paths: {
 }
 ```
 
+Consider the example of overwriting an HTML file in the adminhtml.
+In this example, the `max-length` value of the text-box in the `adminhtml` is altered. The HTML file is located at `vendor/magento/module_ui/view/base/web/templates/form/element/input.html`
+
+-  Create a `requirejs-config.js` file under `app/code/<Vendor_Name>/<Module_Name>/view/base/`
+
+```javascript
+var config = {
+    paths: {
+        'ui/template/form/element/input': '<vendor_name>_<module_name>/template/form/element/input'
+    }
+};
+```
+
+-  Create an `input.html` file under `app/code/<Vendor_Name>/<Module_Name>/view/base/web/template/form/` and copy the contents of the `input.html` file from the `module_ui` template file.
+
+{:.bs-callout-info}
```

Comment:
If the list is numbered then I think this info box should be moved to the end so it doesn't split the list.